.nvmrc

```diff
@@ -1 +1 @@
-v12.18.4
+14
```

Comment:
The move to Next.js didn't require this. Running anything below Node 14 is painful on an M1, so it was bumped up.